assets/sass/hel/variables.scss

```diff
@@ -4,6 +4,7 @@ $light-gray:    #ebedf1;
 $dark-gray:     #525a65;
 $medium-gray:   #abb2bd;
 $black:         #000000;
+$hover-blue:    #0072c6;
```

Comment:
`#0072c6;` this value is the same as `$hel-coat` couldn't that be used here?

Reply:
removed, now using them-info instead